test/parser_test.exs

```diff
@@ -43,12 +46,67 @@ defmodule Prismic.ParserTest do
     test "parses http link fragments" do
       parsed =
         %{type: "Link.web", value: %{url: "https://general.com"}}
-        |> Prismic.Parser.parse_web_link()
+        |> Parser.parse_web_link()
 
-      assert parsed == %Prismic.Fragment.WebLink{
+      assert parsed == %WebLink{
                target: nil,
                url: "https://general.com"
              }
     end
   end
+
+  @prismic_document_link %{
+    type: "Link.document",
+    value: %{
+      document: %{
+        id: "XBLlVREAACsA5_9J",
+        type: "measurement",
+        tags: [],
+        lang: "en-us",
+        slug: "womens-shoulder"
+      },
+      isBroken: false
+    }
+  }
+
+  @parsed_document_link %DocumentLink{
+    broken: false,
+    id: "XBLlVREAACsA5_9J",
+    lang: "en-us",
+    slug: "womens-shoulder",
+    tags: [],
+    target: nil,
+    type: "measurement",
+    uid: nil
+  }
+
+  describe "parse_document_link/1" do
+    test "translates Prismic v1 response format into DocumentLink struct" do
+      assert Parser.parse_document_link(@prismic_document_link) == @parsed_document_link
+    end
+
+    test "includes fragments in DocumentLink when Prismic responseincludes linked data" do
```

Comment:
`response includes` 💅 .

Reply:
I disagree here, or at least I think it's a matter of opinion. I'm used to starting tests with a present-tense verb, thanks to experience with RSpec where tests look like `it "does something"`. It doesn't read quite as nicely with ExUnit but I think it's still ok.

Reply:
I think he means this part "responseincludes"

Reply:
Oh man! Sorry! 🤦‍♂️Thanks

Reply:
Np 😆